elfpy/eth/transactions.py

```diff
@@ -220,6 +222,47 @@ def smart_contract_transact(
         raise err
 
 
+def eth_transact(
```

Comment:
I'd rename this `eth_transfer`, lets' save `eth_transact` for a generic raw eth call function